src/scaladoc/scala/tools/nsc/doc/html/resource/lib/index.css

```diff
@@ -479,6 +479,13 @@ div#content-container > div#content {
   margin: 4em auto 0;
 }
 
+div#content-container > div#content > div#template > div#allMembers > div.members > ol > li > a.anchorToMember {
```

Comment:
should we make this path a little bit less precise? it seems that it could quite easily break when refactoring anything on that path.

it seems to me, a plain `#anchorToMember` would be fine.

Reply:
you are right. i didn't care about refactoring too much.

i'll modify it. however, i think it would be `.anchorToMember`, not `#anchorToMember`.